agimus_controller_ros/agimus_controller_ros/agimus_controller_parameters.yaml

```diff
@@ -80,3 +80,19 @@ agimus_controller_params:
     type: bool
     default_value: True
     description: "Wether we account for self-collision"
+  collision_pairs_names:
+    type: string_array
+    default_value: [""]
+    description: List of names of collision pairs passed to the OCP.
+    read_only: true
+  __map_collision_pairs_names:
+    first:
+      type: string
+      description: First collision pair to map.
+      validation:
+        not_empty<>: []
+    second:
+      type: string
+      description: First collision pair to map.
```

Comment:
Wouldn't it be?
```suggestion
      description: second collision pair to map.
```

Reply:
It should rather be *name of the first/second collision geometry of the collision pair*.